projects/plugins/debug-helper/modules/class-protect-helper.php

```diff
@@ -0,0 +1,354 @@
+<?php
+/**
+ * Jetpack Protect helper class.
+ *
+ * @package automattic/jetpack-debug-helper
+ */
+
+use Automattic\Jetpack\Plugins_Installer;
+use Automattic\Jetpack\Sync\Functions as Sync_Functions;
+
+/**
+ * Helps debug Protect
+ */
+class Protect_Helper {
+
+	/**
+	 * Options.
+	 */
+	const STORED_OPTIONS_KEY = 'protect_helper_option_name';
+
+	/**
+	 * Construction.
+	 */
+	public function __construct() {
+		add_action( 'admin_menu', array( $this, 'register_submenu_page' ), 1000 );
+
+		add_action( 'admin_post_protect_helper_store_options', array( $this, 'admin_post_store_current_options' ) );
+
+		if ( isset( $_GET['show_notice'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			add_action( 'admin_notices', array( $this, 'display_notice' ) );
+		}
+
+		$settings = self::get_stored_settings();
+		if ( $settings['overwrite_status'] ) {
+			// Display a notice when this module overwrites Protect Status.
+			add_action( 'admin_notices', array( $this, 'display_protect_overwritten_notice' ) );
+
+			$option_name      = Automattic\Jetpack\Protect\Status::OPTION_NAME;
+			$option_time_name = Automattic\Jetpack\Protect\Status::OPTION_TIMESTAMP_NAME;
+			add_filter( "option_$option_name", array( $this, 'get_mock_response' ) );
+			add_filter( "option_$option_time_name", array( $this, 'filter_option_time' ) );
+
+			if ( 'error' === $settings['status'] ) {
+				add_filter( 'pre_http_request', array( $this, 'filter_status_fetch' ), 10, 3 );
+			}
+		}
+
+	}
+
+	/**
+	 * Add submenu item.
+	 */
+	public function register_submenu_page() {
+		add_submenu_page(
+			'jetpack-debug-tools',
+			'Protect Helper',
+			'Protect Helper',
+			'manage_options',
+			'protect-helper',
+			array( $this, 'render_ui' ),
+			99
+		);
+	}
+
+	/**
+	 * Renders the UI.
+	 */
+	public function render_ui() {
+		$settings = $this->get_stored_settings();
+		?>
+		<h1>Protect Helper</h1>
+
+		<form action="<?php echo esc_url( admin_url( 'admin-post.php' ) ); ?>" method="post">
+
+		<table class="form-table" role="presentation">
+		<tbody>
```

Comment:
`tbody` and content needs one more level of indentation

Reply:
fixed